Paintroid/src/main/java/org/catrobat/paintroid/tools/ToolReference.kt

```diff
@@ -19,6 +19,6 @@
 package org.catrobat.paintroid.tools
 
 interface ToolReference {
-    fun get(): Tool
+    fun get(): Tool?
```

Comment:
Hmm, maybe I have already asked, but the history doesn't show a thing 😅 
Please check if this ? is necessary

Reply:
`?` here is necessary, that's why I added it back in this PR😅.